### PR TITLE
Safe send

### PR DESCRIPTION
**Pull request by morlic**

### CODE REVIEW

File: lib/oslc_lisp.pl [hunk 1]
```diff
@@ -20,18 +20,28 @@
...
 :- multifile lisp:funct/3.
 
-lisp:funct(send, [ResourceIRI, PostURI], true) :- !,
+call_and_catch(Goal, R) :-
+  catch_with_backtrace((
+      Goal,
+      R = true
+    ),
+    E,
+    ( print_message(error, E),
```
Comment (berezovskyi) [inline lib/oslc_lisp.pl:29]: what's print_message? What's the different from debug?

Comment (likelion) [reply lib/oslc_lisp.pl:29]: `debug` is mainly for topic based printouts using format's notation.
`print_message` is for printing exception messages that can be extended by handlers and integrating various outputs like IDEs

Comment (morlic) [reply lib/oslc_lisp.pl:29]: don't tell anyone, but i just copied from here: https://www.swi-prolog.org/pldoc/doc_for?object=catch_with_backtrace/3

Comment (berezovskyi) [reply lib/oslc_lisp.pl:29]: stealing like an artist!

**Pull request closed without merging**